Source/WebKit/WebProcess/wpe/WebProcessMainWPE.cpp

```diff
@@ -56,6 +56,22 @@ WTF_ALLOW_UNSAFE_BUFFER_USAGE_END
 namespace WebKit {
 using namespace WebCore;
 
+static void registerInspectorResources()
+{
+    const char* dataDir = PKGDATADIR;
+    GUniqueOutPtr<GError> error;
+
+    const char* path = g_getenv("WEBKIT_INSPECTOR_RESOURCES_PATH");
+    if (path && g_file_test(path, G_FILE_TEST_IS_DIR))
+        dataDir = path;
+
+    GUniquePtr<char> gResourceFilename(g_build_filename(dataDir, "inspector.gresource", nullptr));
+    GRefPtr<GResource> gresource = adoptGRef(g_resource_load(gResourceFilename.get(), &error.outPtr()));
+    if (!gresource)
+        g_error("Error loading inspector.gresource: %s", error->message);
+    g_resources_register(gresource.get());
```

Comment:
I don't remember why we have this code duplicated, but I think the idea in WPE is that we don't want to load the resources unless the inspector is actually used.

Reply:
Factoring out the duplicated code into a function is a good idea, though. It would be nice to land the refactoring at least.

Reply:
Fixed.